### PR TITLE
MOD add FAQ

### PR DESCRIPTION
**Pull request by DavidZeLiang**
Add FAQ for load test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -337,3 +337,47 @@ The package mop-utils depends on python which version is no lower than than 3.8.
...
 
+### Q: What is the difference between the load test (perRPS) and load test (perConcurrency)?
+- **perRPS**: 
+
+    PerRPS is a type of load test that focuses on testing the system's capacity to handle a given number of 
+requests per second. This type of testing is particularly useful when you want to identify the maximum number of 
+requests that the system can handle without degrading its performance. 
+
+    The platform uses python async mechanism to send requests to the service, gradually increasing the "target RPS" from 
+1 using step 10. For each target RPS, the platform runs the load test for 5 minutes, and the requests are randomly sent.
```
Comment (bowgong) [inline README.md:348]: Will this be 1, 11, 21? or 1, 10, 20, ...?

Comment (bowgong) [inline README.md:348]: What's 'randomly' means here?

File: README.md [hunk 2]
```diff
@@ -337,3 +337,47 @@ The package mop-utils depends on python which version is no lower than than 3.8.
...
+    PerConcurrency is a type of load test that focuses on testing the system's capacity to handle a given number of 
+concurrent users. This type of testing is particularly useful when you want to identify the maximum number of users 
+that the system can handle without degrading its performance.
+
+    The platform gradually increases the "concurrency count" from 1, using step 10. For each concurrency count, the 
+platform uses the specified number of single-thread clients to send requests. For example, if the concurrency count 
+is 3, the platform will have 3 single-thread clients, and each client will send requests continuously. Each client 
+is independent of each other, sends a request, gets the response, and then sends the next request with no waiting time.
+
+    The key metrics provided by the platform include the actual concurrency count, latency percentile, successful request
```
Comment (bowgong) [inline README.md:374]: `actual concurrency count` per second?

File: README.md (hunk 1 above)
Comment (DavidZeLiang) [reply README.md:348]: 1,10,20,.... how to describe such a sequence...

Comment (DavidZeLiang) [reply README.md]: add detailed implementation delow.

File: README.md (hunk 2 above)
Comment (DavidZeLiang) [reply README.md]: Typo... fixed.

**Pull request merged**